[2025-07-24T00:01:42.382+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: sales_batch_etl.load_to_postgres scheduled__2025-07-23T00:00:00+00:00 [queued]>
[2025-07-24T00:01:42.388+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: sales_batch_etl.load_to_postgres scheduled__2025-07-23T00:00:00+00:00 [queued]>
[2025-07-24T00:01:42.389+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-07-24T00:01:42.396+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): load_to_postgres> on 2025-07-23 00:00:00+00:00
[2025-07-24T00:01:42.401+0000] {standard_task_runner.py:60} INFO - Started process 23256 to run task
[2025-07-24T00:01:42.404+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'sales_batch_etl', 'load_to_postgres', 'scheduled__2025-07-23T00:00:00+00:00', '--job-id', '30', '--raw', '--subdir', 'DAGS_FOLDER/sales_batch_etl.py', '--cfg-path', '/tmp/tmpu40i62ty']
[2025-07-24T00:01:42.407+0000] {standard_task_runner.py:88} INFO - Job 30: Subtask load_to_postgres
[2025-07-24T00:01:42.434+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-07-24T00:01:42.478+0000] {task_command.py:423} INFO - Running <TaskInstance: sales_batch_etl.load_to_postgres scheduled__2025-07-23T00:00:00+00:00 [running]> on host e17dc2585651
[2025-07-24T00:01:42.589+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='sales_batch_etl' AIRFLOW_CTX_TASK_ID='load_to_postgres' AIRFLOW_CTX_EXECUTION_DATE='2025-07-23T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-07-23T00:00:00+00:00'
[2025-07-24T00:01:42.590+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-07-24T00:01:42.592+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python3 /opt/***/scripts/load_to_postgres.py']
[2025-07-24T00:01:42.600+0000] {subprocess.py:86} INFO - Output:
[2025-07-24T00:01:44.020+0000] {subprocess.py:93} INFO - üì§ Loading data from /tmp/sales.csv to PostgreSQL...
[2025-07-24T00:01:44.032+0000] {subprocess.py:93} INFO - üìä Read DataFrame with shape: (10000, 6)
[2025-07-24T00:01:44.032+0000] {subprocess.py:93} INFO - üõ†Ô∏è Creating table if not exists...
[2025-07-24T00:01:44.032+0000] {subprocess.py:93} INFO - üì• Inserting rows into sales_staging table...
[2025-07-24T00:01:44.032+0000] {subprocess.py:93} INFO - ‚úÖ Data inserted successfully into PostgreSQL.
[2025-07-24T00:01:44.081+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-07-24T00:01:44.110+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=sales_batch_etl, task_id=load_to_postgres, execution_date=20250723T000000, start_date=20250724T000142, end_date=20250724T000144
[2025-07-24T00:01:44.162+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-07-24T00:01:44.185+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
